Assets/Scripts/GraphArranger.cs

```diff
@@ -50,27 +51,25 @@ public void DisableArrangement()
     void Repel()
     {
       for (var i = 0; i < _graphService.Graph.nodes.Count; i++)
-      for (var j = i + 1; j < _graphService.Graph.nodes.Count; j++)
       {
-        var node1 = _graphService.Graph.nodes[i];
-        var node2 = _graphService.Graph.nodes[j];
+        for (var j = i + 1; j < _graphService.Graph.nodes.Count; j++)
```

Comment:
Personally I would rather keep double loops without the brackets

Reply:
i like brackets in this case